po/es.po

```diff
@@ -5014,104 +5287,117 @@ msgid ""
 "various events. Use `#[derive(Debug)]` to allow the types to be formatted "
 "with `{:?}`."
 msgstr ""
+"Crearemos una estructura de datos para representar un evento en un sistema "
+"de control de ascensores. Debes definir los tipos y las funciones para crear "
+"varios eventos. Usa "
 
+# [derive(Debug)]" para permitir que se aplique el formato `{:?}` a los tipos."
```

Comment:
Please fix this. It has been accidentally split from above because of the quotes and the use of #

My best suggestion is something along these lines:
```suggestion
"`#[derive(Debug)]` para permitir que se aplique el formato `{:?}` a los tipos."
```

But you also need to do some clean up on the previous line.